### PR TITLE
Make ortho projection more discoverable and easier to use

### LINKED ISSUE #5123: Make ortho projection more discoverable and easier to use
@bagnell can you please do this next to polish #5021?

* [x] Add an "Orthographic Projection" Sandcastle example that creates the Projection Picker widget and perhaps loads a glTF model or creates a box geometry.
* [x] Is it possible to move the `_switchToPerspective` and `_switchToOrthographic` functions from the [widget](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Widgets/ProjectionPicker/ProjectionPickerViewModel.js#L116-L179) to a more generic place so that apps using their own widgets can easily switch between projections?  This is similar to decoupling the morph from the scene pick widget and decoupling the geocoder from the geocoder widget.

Part of #4678

### PR DESCRIPTION
**Pull request by bagnell**
Fixes #5123.

### CODE REVIEW

**Comment by lilleyse** [comment]
Since the app starts in ortho, should the widget also be at the ortho state?

File: Apps/Sandcastle/gallery/Projection.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Change the projection between perspective and orthographic.">
+    <meta name="cesium-sandcastle-labels" content="Tutorials,Showcases">
```
Comment (lilleyse) [inline Apps/Sandcastle/gallery/Projection.html]: Should this be in the Tutorials section?

**Comment by bagnell** [comment]
@lilleyse This is ready for another look.

**Comment by lilleyse** [comment]
Looks good.

**Comment by pjcozzi** [comment]
Nice, please merge into `3d-tiles`.

**Pull request merged**